pyxis_slurmstepd.c

```diff
@@ -86,17 +90,20 @@ static struct plugin_context context = {
 	.job = {
 		.uid = -1, .gid = -1, .privileged = false,
 		.jobid = 0, .stepid = 0,
-		.local_task_count = 0, .total_task_count = 0,
+		.local_task_count = 0, .total_task_count = 0, .nodeid = 0,
```

Comment:
nit: if you don't specify values in struct initializer it will be 0. maybe it's a convention here

Reply:
keep it as convention :)